### PR TITLE
Rename quickstart (after initial feedback)

### PR DESCRIPTION
**Pull request by JimHagan**
# Summary

After some early feedback from the field I renamed this quickstart to `nrql-query-performance` to make it more positive and a little easier to find.

### CODE REVIEW

File: quickstarts/nrql-query-performance/config.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 id: 47436e4a-f74c-4d87-8756-afc203a0f0d9
-slug: slow-nrql-query-analytics
+slug: nrql-query-performance
 title: Slow Query Analytics
```
Comment (sjyothi54) [inline quickstarts/nrql-query-performance/config.yml:3]:
```suggestion
title: NRQL Query Performance
```

**Review by sjyothi54 — CHANGES_REQUESTED** [review]
@JimHagan Could you please modify the title as well?

**Comment by JimHagan** [comment]
> @JimHagan Could you please modify the title as well?

Done.  Thanks for catching that.

**Review by sjyothi54 — APPROVED** [review]
LGTM

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM

**Pull request merged**